ldtkgo.go

```diff
@@ -6,24 +6,26 @@ package ldtkgo
 import (
 	"encoding/json"
 	"errors"
+	"fmt"
 	"image"
 	"image/color"
 	"io/ioutil"
+	"os"
 	"path/filepath"
 	"strconv"
 
 	"github.com/tidwall/gjson"
 )
 
-//LayerType constants indicating a Layer's type.
+// LayerType constants indicating a Layer's type.
```

Comment:
Some changes like this are due to the `gofmt` on-save.
I hope it doesn't make this PR much harder to review.

It would be better to send them separately (if at all), maybe?